### PR TITLE
fix: allow empty field titles in decrypted responses

### PR DESCRIPTION
**Pull request by justynoh**
## Problem

On 14 Nov 2022, a bug was identified in FormSG that allowed admins to create fields with empty field titles via the UI. As an unintended consequence, form responses to such storage mode forms became undecryptable. However, this was not caught to be an issue at the time, and thus was not communicated to form admins. This issue only came to light on 14 Dec 2022, when an admin of an affected form ended up not being able to decrypt form responses.

## Solution

This PR changes the decrypted response validator to check for the type of the field title, in order to allow for empty strings. It also adds a test for this in the spec.

### CODE REVIEW

File: spec/crypto.spec.ts [hunk 1]
```diff
@@ -122,6 +123,21 @@ describe('Crypto', function () {
     expect(decrypted).toHaveProperty('responses', plaintextMultiLang)
   })
 
+  it('should be able to encrypt and decrypt submissions with empty field titles from 2022-11-14 end-to-end successfully', () => {
```
Comment (karrui) [inline spec/crypto.spec.ts:126]: Don't need so specific test la haha, just say can decrypt empty strings

**Review by karrui — APPROVED** [review]
Lgtm

File: spec/crypto.spec.ts (hunk 1 above)
Comment (justynoh) [reply spec/crypto.spec.ts:126]: hahah oops i follow format only xD
